src/pybind/mgr/cephadm/serve.py

```diff
@@ -529,20 +529,23 @@ def _apply_service(self, spec: ServiceSpec) -> bool:
         svc = self.mgr.cephadm_services[service_type]
         daemons = self.mgr.cache.get_daemons_by_service(service_name)
 
-        public_network = None
+        public_networks = None
```

Comment:
```suggestion
        public_networks: List[str] = []
```

We no longer need a None value here

Reply:
Thanks Sebastian! Added your suggestion.